samples/cpp/speculative_decoding_lm/speculative_decoding_lm.cpp

```diff
@@ -26,15 +26,11 @@ int main(int argc, char* argv[]) try {
     // Please, set device for main model in `LLMPipeline` constructor and in in `ov::genai::draft_model` for draft.
     std::string main_device = "CPU", draft_device = "CPU";
 
-    ov::genai::SchedulerConfig scheduler_config;
-    scheduler_config.cache_size = 5;
```

Comment:
we have recently merged prompt lookup decoding via PA, which uses the same code:
https://github.com/openvinotoolkit/openvino.genai/blob/9bcadf7ffdcfe5b133605847d964759593949fac/samples/cpp/prompt_lookup_decoding_lm/prompt_lookup_decoding_lm.cpp#L25-L26

Reply:
Removed `scheduler_config` and setting of `cache_size`, the rest left as is.